### DISCUSSION TITLE
How to customise the style of TextEdit/LineEdit?

### DISCUSSION THREAD

**Discussion by arunpkio** [slint-ui/slint#6029] [Q&A] [answered]
I couldn't find property `background`/`border.color` for TextField to override the default colors.
Is there a way to change the default colors of these standard widgets? 
If not is there an alternative to create custom TextField component t have a custom style?

**Comment by Horbin-Magician** [comment, marked as answer]
In my opinion, you can use a lower-level item, TextInput, https://releases.slint.dev/1.7.2/docs/slint/src/language/builtins/elements.html#textinput.
You can coat it with other components for custom purposes.

**Reply by arunpkio** [reply]
Thank you, that was helpful. I somehow missed it.
